From 6e382246582edbf501a5fe65628867aaed270bca Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Mon, 29 Oct 2018 14:00:11 +0800
Subject: [PATCH 4942/5242] MLK-20110 drm/imx: dpu: crtc: Add pclk and
 vrefresh dbg info in ->mode_set_nofb

commit  c441ced3f1a8949e572dd6f49d514f2cf540d3d9 from
https://source.codeaurora.org/external/imx/linux-imx.git

Pixel clock and vrefresh are important information for a video mode.
This patch adds debug information for them in ->mode_set_nofb.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/dpu/dpu-crtc.c |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/gpu/drm/imx/dpu/dpu-crtc.c b/drivers/gpu/drm/imx/dpu/dpu-crtc.c
index e3ad4d8..1bc701b 100644
--- a/drivers/gpu/drm/imx/dpu/dpu-crtc.c
+++ b/drivers/gpu/drm/imx/dpu/dpu-crtc.c
@@ -756,6 +756,10 @@ static void dpu_crtc_mode_set_nofb(struct drm_crtc *crtc)
 			mode->hdisplay);
 	dev_dbg(dpu_crtc->dev, "%s: mode->vdisplay: %d\n", __func__,
 			mode->vdisplay);
+	dev_dbg(dpu_crtc->dev, "%s: mode->clock: %dKHz\n", __func__,
+			mode->clock);
+	dev_dbg(dpu_crtc->dev, "%s: mode->vrefresh: %dHz\n", __func__,
+			drm_mode_vrefresh(mode));
 	if (dcstate->use_pc)
 		dev_dbg(dpu_crtc->dev, "%s: use pixel combiner\n", __func__);
 
-- 
1.7.9.5

